protocol-samples/http-client-to0-sample/src/main/java/org/fido/iot/sample/To0ClientApp.java

```diff
@@ -275,12 +276,21 @@ private void run(String[] args)
     DispatchResult dr = clientService.getHelloMessage();
 
     Composite ovh = clientStorage.getVoucher().getAsComposite(Const.OV_HEADER);
+
+    UUID guid = ovh.getAsUuid(Const.OVH_GUID);
+    System.out.println("TO0 with guid " + guid.toString());
+
     Composite rvi = ovh.getAsComposite(Const.OVH_RENDEZVOUS_INFO);
 
-    List<String> paths = RendezvousInfoDecoder.getHttpDirectives(
+    List<String> rviPaths = RendezvousInfoDecoder.getHttpDirectives(
         rvi,
         Const.RV_OWNER_ONLY);
 
+    List<String> paths = new ArrayList<>();
+    for (String path : rviPaths) {
+      paths.add(path.replaceAll("https://", "http://"));
```

Comment:
Why are we removing https for TO0?  Shouldn't TO0 be https only?

Reply:
https is not supported in protocol-samples.